hive-metastore/src/test/java/org/apache/iceberg/hive/TestLoadHiveCatalog.java

```diff
@@ -24,23 +24,23 @@
 import org.apache.iceberg.CatalogProperties;
 import org.apache.iceberg.CatalogUtil;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
-import org.junit.AfterClass;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.assertj.core.api.Assertions;
```

Comment:
assertThat can be statically imported for files where it hasn't been previously used

Reply:
static import was added for assertThat